26-register_registration/resource_types/artistic_result/creator_selects_resource_type_artistic_result_and_subtype_architecture.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Creator selects Resource type Artistic Result and subtype Architecture
+
+  @3413
+  Scenario: Creator navigates to the Resource Type tab and selects Resource subtype "Architecture"
+    Given Creator navigates to the Resource Type tab and selects Resource type "Artistic Result"
```

Comment:
I suspect that the way these acceptance reqs should be written is:

Given the creator wants to add an Artistic Result - Architecture
Then they can add information about:
      | Construction           |
      | Plan proposal          |
      | Landscape architecture |
      | Interior               |
      | Other                  |
And they can add Exhibitions of type:
      | Competition            |
      | Publication or Mention |
      | Prize or Award         |
      | Exhibition             |
And they can edit existing Exhibitions
And they can delete existing Exhibitions

Reply:
I think we need a general discussion about this, though.